.travis.yml

```diff
@@ -1,16 +1,44 @@
 language: ruby
 rvm:
+  - 2.6
+  - 2.5
+  - 2.4
+  - 2.3
   - 2.2
   - 2.1
 cache: bundler
-before_install: gem install bundler
+before_install: gem install bundler -v 1.16.2
```

Comment:
I'm okay constraining this, but a comment explaining why would be nice so we can investigate and remove it later

Reply:
I've explained in the commit message, which I think is actually better than a comment because it's less likely to become wrong as the code changes. Is that sufficient?